keras_cv/layers/preprocessing/solarization.py

```diff
@@ -49,16 +58,31 @@ class Solarization(tf.keras.layers.Layer):
             or [height, width, channels].
     """
 
-    def __init__(self, threshold=None):
+    def __init__(
+        self,
+        addition=0.0,
+        threshold=None,
+        value_range=(0, 255),
+    ):
         super().__init__()
+        self.addition = addition
         self.threshold = threshold
+        self.value_range = value_range
 
-    def call(self, images):
-        images = tf.clip_by_value(images, clip_value_min=0, clip_value_max=255)
+    def augment_image(self, image, transformation=None):
+        image = preprocessing.transform_value_range(
+            image, original_range=self.value_range, target_range=(0, 255)
+        )
+        result = image + self.addition
+        result = tf.clip_by_value(result, 0, 255)
         if self.threshold is None:
```

Comment:
if we have the default value to be 0, then we can actually remove this condition and always use the _solarize_above_threshold.

Reply:
Good point, fixed